gluten-flink/docs/Flink.md

```diff
@@ -91,24 +91,51 @@ Submit test script from `flink run`. You can use the `StreamSQLExample` as an ex
 ### Flink local cluster
 
 After deploying flink binaries, please add gluten-flink jar to flink library path,
-including gluten-flink-runtime-1.4.0.jar, gluten-flink-loader-1.4.0.jar and Velox4j jars above.
+including gluten-flink-runtime-1.5.0.jar, gluten-flink-loader-1.5.0.jar and Velox4j jars above.
+Take following command as an example:
+
+```shell
+# notice: first set your own specified project home, you may set it mannualy in you .bash_profile so that it can auto take effect. 
+export VELOX4J_HOME=
+export GLUTEN_FLINK_HOME=
+export FLINK_HOME=
+
+cd $FLINK_HOME
+mkdir -p gluten_lib
+ln -s $VELOX4J_HOME/target/velox4j-0.1.0-SNAPSHOT.jar $FLINK_HOME/gluten_lib/velox4j-0.1.0-SNAPSHOT.jar
+ln -s $GLUTEN_FLINK_HOME/runtime/target/gluten-flink-runtime-1.5.0-SNAPSHOT.jar $FLINK_HOME/gluten_lib/gluten-flink-runtime-1.5.0.jar
+ln -s $GLUTEN_FLINK_HOME/loader/target/gluten-flink-loader-1.5.0-SNAPSHOT.jar $FLINK_HOME/gluten_lib/gluten-flink-loader-1.5.0.jar
+```
+
 And make them loaded before flink libraries.
+
+#### How to make sure gluten packages loaded first in Flink?
+
+Gluten packages need to be loaded first in Flink, you can edit the `bin/config.sh` in Flink dir,
+and change the `constructFlinkClassPath` function like this:
```

Comment:
Let's make it concise, suggestion:

*** you can modify the `constructFlinkClassPath` function in `$FLINK_HOME/bin/config.sh` like this: